[2024-10-17T17:20:43.887+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T17:20:43.920+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.monetized_stock scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T17:20:43.937+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.monetized_stock scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T17:20:43.939+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T17:20:43.975+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): monetized_stock> on 2024-10-16 00:00:00+00:00
[2024-10-17T17:20:43.986+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=147) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T17:20:43.989+0000] {standard_task_runner.py:72} INFO - Started process 151 to run task
[2024-10-17T17:20:43.991+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'monetized_stock', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmpg9w9zkac']
[2024-10-17T17:20:43.996+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask monetized_stock
[2024-10-17T17:20:44.086+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.monetized_stock scheduled__2024-10-16T00:00:00+00:00 [running]> on host 1652cc614667
[2024-10-17T17:20:44.312+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='monetized_stock' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T17:20:44.314+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T17:20:44.683+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: monetize_stock() missing 2 required positional arguments: 'stock_file' and 'quotes_file'
[2024-10-17T17:20:44.706+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_update_stock_ferrimac, task_id=monetized_stock, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T172043, end_date=20241017T172044
[2024-10-17T17:20:44.736+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T17:20:44.738+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 11 for task monetized_stock (monetize_stock() missing 2 required positional arguments: 'stock_file' and 'quotes_file'; 151)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: monetize_stock() missing 2 required positional arguments: 'stock_file' and 'quotes_file'
[2024-10-17T17:20:44.769+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-17T17:20:44.809+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-17T17:20:44.829+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-17T22:09:32.775+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T22:09:32.806+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.monetized_stock scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T22:09:32.822+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.monetized_stock scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T22:09:32.823+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T22:09:32.846+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): monetized_stock> on 2024-10-16 00:00:00+00:00
[2024-10-17T22:09:32.856+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=181) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T22:09:32.859+0000] {standard_task_runner.py:72} INFO - Started process 187 to run task
[2024-10-17T22:09:32.860+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'monetized_stock', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmpyq125jof']
[2024-10-17T22:09:32.865+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask monetized_stock
[2024-10-17T22:09:32.969+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.monetized_stock scheduled__2024-10-16T00:00:00+00:00 [running]> on host d95e36bcd28a
[2024-10-17T22:09:33.105+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='monetized_stock' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T22:09:33.107+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T22:09:36.130+0000] {warnings.py:112} WARNING - /opt/***/functions_etl/data_transform.py:31: UserWarning: Could not infer format, so each element will be parsed individually, falling back to `dateutil`. To ensure parsing is consistent and as-expected, please specify a format.
  df_stock.columns = pd.to_datetime(df_stock.columns, errors='ignore', dayfirst=True)

[2024-10-17T22:09:36.139+0000] {warnings.py:112} WARNING - /opt/***/functions_etl/data_transform.py:33: UserWarning: Could not infer format, so each element will be parsed individually, falling back to `dateutil`. To ensure parsing is consistent and as-expected, please specify a format.
  df_list_prices.columns = pd.to_datetime(df_list_prices.columns, errors='ignore', dayfirst=True)

[2024-10-17T22:09:37.038+0000] {logging_mixin.py:190} INFO - Stock file has been updated for date 2024-10-16 00:00:00.
[2024-10-17T22:09:37.041+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-17T22:09:37.061+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T22:09:37.063+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=monetized_stock, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T220932, end_date=20241017T220937
[2024-10-17T22:09:37.132+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T22:09:37.147+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
